.github/dependabot.yml

```diff
@@ -12,8 +12,7 @@ updates:
     ignore:
       - dependency-name: '*'
         update-types: ['version-update:semver-patch'] # Ignore all patch updates for version updates only
-      # We are looking at updating Storybook, so don't tell us about it. Remove this once we've upgraded.
-      - dependency-name: '@storybook*'
+      # We are looking at updating Chromatic, so don't tell us about it. Remove this once we've upgraded.
```

Comment:
Could we have a ticket created for this in the style of https://github.com/wellcomecollection/wellcomecollection.org/issues/11228#issue-2569977526 (with the reminder to remove this ignore) 🙏